jumpscale/sals/vdc/deployer.py

```diff
@@ -140,7 +140,7 @@ def transaction_hashes(self):
 
     @transaction_hashes.setter
     def transaction_hashes(self, value):
-        self.info(f"adding transactions {value}", disable_bot=True)
+        self.info(f"adding transactions {value}, vdc uuid: {self.vdc_uuid}", disable_bot=True)
```

Comment:
No need to add vdc uuid, it's already in the message here https://github.com/threefoldtech/js-sdk/blob/development/jumpscale/sals/vdc/deployer.py#L100
and should be applied in all logs in the deployer

Reply:
indeed, yet it's called vdc only we can add uuid to it :D